app/views/communicarts/approval_response.html.haml

```diff
@@ -1,7 +0,0 @@
-!!!
-%head
-  %title= "Communicart: Approval"
-%html
-
-  %body.communicart-notification
-= render partial: "carts/cart", locals: {cart: @cart, title: "Cart #{@cart.id}"}
```

Comment:
I prefer haml, but the general consensus from other has been to kill it. I went ahead and converted since I was in the template.